Sloth.Web/Startup.cs

```diff
@@ -183,6 +183,13 @@ public void ConfigureServices(IServiceCollection services)
                         })
                     });
 
+                routes.MapRoute(
```

Comment:
This works, but I don't really like having a special map route for this. I'm OK with it, but when @srkirkland gets back lets see what he says.